contracts/truefi2/PoolFactory.sol

```diff
@@ -50,18 +54,33 @@ contract PoolFactory is IPoolFactory, UpgradeableClaimable {
      */
     event PoolCreated(address token, address pool);
 
+    /**
+     * @dev Event to show creation of the new private pool
+     * @param borrower Address of new pool's borrower
+     * @param token Address of token, for which the pool was created
+     * @param pool Address of new pool's proxy
+     */
+    event PrivatePoolCreated(address borrower, address token, address pool);
+
     /**
      * @dev Event to show that token is now allowed/disallowed to have a pool created
      * @param token Address of token
      * @param status New status of allowance
      */
-    event AllowedStatusChanged(address token, bool status);
+    event TokenAllowedStatusChanged(address token, bool status);
```

Comment:
I think we should never change event names. We have them to have a clear history of what was happening with the contract and changing it could be confusing. On the other hand it sucks to have AllowedStatusChanged and BorrowerAllowedStatus changed. Maybe we could use the whitelist term for borrowers instead of Allowance?